### DISCUSSION TITLE
How to monitor the EdgeDB (or the underlying postgres instance)?

### DISCUSSION THREAD

**Discussion by isidentical** [geldata/gel#1823] [Q&A] [answered]
I have a grafana setup (w/prometheus), is there an official way of connecting EdgeDB (or the underlying postgres clusters) as datasources to there?

**Comment by elprans** [comment, marked as answer]
There is a plan for a proper metrics endpoint via a specially-configured `Port` (which could also double as a healthcheck endpoint).  That is blocked on #1460, which would happen before 1.0 is out.

If you have specific requirements or ideas regarding this, feel free to share!

**Reply by isidentical** [reply]
Cool, thanks for the information. I dont have any special requirements.

**Comment by ofek** [comment]
Any update on this?

**Reply by clmnin** [reply]
You could start here [EdgeDB Grafana Frontend](https://github.com/edgedb/edgedb-grafana-frontend)
